velox/dwio/common/Options.cpp

```diff
@@ -70,4 +70,11 @@ std::string_view toString(FileFormat fmt) {
   }
 }
 
+void updateColumnOptionsFromReaderOptions(
```

Comment:
Could be just `ColumnReaderOptions fromReaderOptions(const ReaderOptions& options)`

Reply:
Thanks for the suggestion. Updated.